agenta-cli/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "agenta"
-version = "0.28.0"
+version = "0.28.2a1"
```

Comment:
might want to remove this before merging.

Reply:
Yup, thank you.